pkg/routes/logs_test.go

```diff
@@ -0,0 +1,45 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestPreCheckLogFileNameLength(t *testing.T) {
+	logFilePath := "/var/log"
+	oversizeFileName := fmt.Sprintf("%s/%0256s", logFilePath, "a")
+	correctFileName := fmt.Sprintf("%s/%0255s", logFilePath, "a")
+
+	isOversize := preCheckLogFileNameLength(oversizeFileName)
+	if !isOversize {
+		t.Error("failed to check oversize filename")
+	}
+
+	NotOversize := preCheckLogFileNameLength(correctFileName)
```

Comment:
the file is not created, hence may not exist at all, that could cause the 'false' return

Reply:
As I test in here, http.ServeFile will return code 500 to oversize filename, and 500 is not the code that we want here, so I did a preCheckLogFileNameLength in advance.
It's only designed to check filename length, and return true when given file path contain oversize(more than 255) string. When the file is not exist, error won't be catch and will continue to execute the old process(http.ServeFile process them).
Are you saying I should handle other err like "file not exist" in precheck?

Reply:
no, i'm not suggesting to handle other error. 

i'm trying to say the test case for correctFileName is likely not really testing the intended condition. For correctFileName case, inside preCheckLogFileNameLength, os.Stat(filePath) will return error os.ErrNotExist

Reply:
Sorry, I get it now.
I was missing the test case for existing correct file, and I add one below that create the file first.